src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Tokens/ConfigurationBasedIssuerNameRegistry.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System;
+
+namespace CoreWCF.IdentityModel.Tokens
+{
+    public class ConfigurationBasedIssuerNameRegistry : IssuerNameRegistry
+    {
```

Comment:
This class doesn't make sense to include as it's an IssuerNameRegistry implementation which is populated from your app.config/web.config. We should look to add some support for making this asp.net core config driven, but out of scope. Can you create an issue for tracking that?

Reply:
can we keep this as it helps me to test ?